index.js

```diff
@@ -6,24 +6,24 @@ gameStartElement.addEventListener('click', (event) => {
 
 document.addEventListener("keydown", event => {
     if (event.isComposing || event.keyCode === 40) {
-            console.log('KeyDown is pressed')
+        console.log('KeyDown is pressed')
     }
 })
 
 document.addEventListener("keydown", event => {
     if (event.isComposing || event.keyCode === 38) {
```

Comment:
If `event.isComposing` is true, then all events (left, right, top, down) are going to be fired. This looks like an error.